stories/src/index.stories.tsx

```diff
@@ -88,6 +104,22 @@ storiesOf('XY Plot', module)
     const x = xKnob(table)
     const y = yKnob(table)
     const timeZone = timeZoneKnob()
+    const timeFormat = select(
+      'Time Format',
+      {
+        'DD/MM/YYYY HH:mm:ss.sss': 'DD/MM/YYYY HH:mm:ss.sss',
+        'MM/DD/YYYY HH:mm:ss.sss': 'MM/DD/YYYY HH:mm:ss.sss',
+        'YYYY/MM/DD HH:mm:ss': 'YYYY/MM/DD HH:mm:ss',
+        'hh:mm a': 'hh:mm a',
```

Comment:
have we added this option onto the influxdb formatters or did we replace the `HH:mm a`?

Reply:
After giraffe's next release, i will add this to influxdb and bump giraffe at the same time.